bundles/org.openhab.ui/web/src/components/config/controls/item-picker.vue

```diff
@@ -10,7 +10,8 @@
       <f7-button slot="media" icon-f7="list_bullet_indent" @click.native="pickFromModel" />
     </f7-list-item>
     <!-- for placeholder purposes before items are loaded -->
-    <f7-list-item link v-show="!ready" :title="title">
+    <f7-list-item link v-else title="Loading...">
```

Comment:
```suggestion
    <f7-list-item link v-show="!ready" :title="title" disabled no-chevron>
```
This is more subtle and doesn't break the item picker's "smart select" settings - it opens in a page instead of a popup and doesn't work at all when called from a popup, probably due to the VDOM patching.

Reply:
I'd like to learn, can you please explain why `v-show` here and not `v-else`?

Reply:
Essentially because of https://vuejs.org/v2/guide/conditional.html#Controlling-Reusable-Elements-with-key.
Vue here would try to reuse the non-functional "loading" list item for the actual one and this leads to problems notably the smart select options not being considered. Another option would have been to add `key` attributes to both alternatives to prevent this re-using.

Reply:
ah, ok, thank you!